source/manual/manage-assets.html.md.erb

```diff
@@ -60,71 +80,108 @@ not provide the facility for editors to upload assets such as images and PDFs.
 In these rare cases, we can upload assets to asset-manager manually and give
 the URL to content editors to embed.
 
-Production assets are replicated to staging and integration nightly, so it is
-best to simply perform the upload directly in production. First, upload the
-asset to a backend machine:
+> Production assets are replicated to staging and integration nightly, so it is
+best to simply perform the upload directly in production.
 
-```sh
-$ gds govuk connect scp-push -e production aws/backend:1 my_file.jpg /tmp
-```
+> If the pod terminates at any point during this process (e.g. a deployment occurs),
+> you will need to start again as the `/tmp` directory is specific to the pod.
 
-Then SSH to the same machine and run the upload command:
+1. Get the name of the asset-manager pod (you need the app pod, which will be named
+   like `asset-manager-abc1234-abc123` and does not contain any extra words, such as
+   `freshclam`, `pss` or `worker`):
```

Comment:
We could put this information in the grep, but it is perhaps less approachable 

```
kubectl get pods --all-namespaces | grep asset-manager | grep -v -e freshclam -e pss -e worker
```

Reply:
I did consider this, but decided it's not a definitive list as it's possible pods with other names could be added in time.

Reply:
Agreed!